ITE/1/README.adoc

```diff
@@ -0,0 +1,925 @@
+====
+
+=== ITE Types
+
+There are three kinds of ITE:
+
+. A **Standards Track** ITE describes a new feature or implementation
+ for the in-toto specification. It may also describe an interoperability or
+ backwards-compatibility standard which will be supported for the feature in
+ current versions of in-toto, moving forward.
+. An **Informational** ITE describes an in-totodesign issue, or
+ provides general guidelines or information to the in-toto community,
+ but does not propose a new feature. Informational ITEs do not
+ necessarily represent a in-toto community consensus or
+ recommendation, so users and implementers are free to ignore
+ Informational ITEs or follow their advice. An example of an informational ITE 
+ would be the recommendations for integrating in-toto and TUF.
+. A **Process** ITE describes a process surrounding the in-toto project, or
+ proposes a change to (or an event in) a process. Process ITEs are like
+ Standards Track ITEs but apply to areas other than the in-toto specification
+ itself. They may propose an implementation, but not for what would be
+ generally considered the in-toto codebase; they often require community
+ consensus; unlike Informational ITEs, they are more than recommendations, and
+ users are typically not free to ignore them. Examples include procedures,
+ guidelines, changes to the decision-making process, and changes to the tools
+ or environment used in in-toto development. Any meta-ITE (ITE focusing on the
+ improvement of the in-toto Enhancement process) is also considered a
+ Process ITE.
+
+=== ITE Terminology
+
+==== Editor
+
+The ITE editors are individuals responsible for managing the administrative
+and editorial aspects of the ITE workflow (e.g. assigning ITE numbers and
+changing their status). See <<editor-responsibilities, ITE Editor Responsibilities & Workflow>> for
+details. 
+
+ITE editorship is by invitation of the current editors. All of the ITE workflow
+can be conducted via the GitHub ITE Repository footnoteref:[repo] and pull
+requests. Editors are members of a GitHub group called `ITE-editors`.
+
+==== Contributor
+
+A ITE may have any number of `Contributors` who help write, implement, discuss,
+or offer feedback about that ITE. One contributor might do only one or any
+combination of these of these things during any part of the life of a ITE.
+While we invite contributions by companies or other organizations, contributors
+listed in a ITE need to be specific people.
+
+==== Sponsor
+
+Each ITE has at least one "Sponsor".
+
+The ITE Sponsor is a contributor who is responsible for the ITE throughout its
+lifecycle. Their responsibilities include:
+
+* Building consensus for that ITE within the community
+* Documenting dissenting opinions
+* Coordinating contributors' work
+* Ensuring the ITE meets the style, format, and quality guidelines
+* Maintaining the ITE after it is finalized
+* Setting and communicating the schedule as needed
```

Comment:
Not clear what this means, i.e. what "schedule" does it refer to.

Reply:
As to when reviews are due. I tried to rephrase to make this clearer